### PR TITLE
Feat: Made alpha-server deployable 

### PR DESCRIPTION
**Pull request by MartinSchmidt**
I have made the alpha-server deployable.

NOTE: It always runs in memory with two registries, it will be made configurable in the near future.

- Created dockerfile to create server
- Added server release to workflow
- Updated workflow to new output syntax
- Changed record options to classes 
- Restructured VerifiableEventStore BatchProcessor.
- Configured server to start two in-memory event stores.

### CODE REVIEW

File: src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs [hunk 1]
```diff
@@ -17,8 +17,8 @@ public async Task Will_Finalize_Full_Batch()
         var blockId = new Fixture().Create<string>();
         var transactionId = new Fixture().Create<string>();
         var blockChainConnector = new Mock<IBlockchainConnector>();
-        var batcherOptions = new BatcherOptions() { BatchSizeExponent = 2 };
-        var eventStore = new MemoryEventStore(batcherOptions);
+        var batcherOptions = new VerifiableEventStoreOptions() { BatchSizeExponent = 2 };
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs:20]: Is the BatchSizeExponent hardcoded here ?!? or is the = the default value

**Review by wisbech — APPROVED** [review]
Great work as always.

Looks to me that a config-options file might be worthwhile to control the size of the merkle-tree. At this point it is nice-to-have but would be meaningful in the long run.

File: src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs (hunk 1 above)
Comment (MartinSchmidt) [reply src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs:20]: it is a test case, but I just saw we have negative tests, I will add some more.

Comment (MartinSchmidt) [reply src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs]: changed it to a  theory, with multiple test sweeps

**Review by wisbech — APPROVED** [review]
Alpha ready to go

**Pull request merged**